server/ai_live_video.go

```diff
@@ -228,6 +238,45 @@ func startTrickleSubscribe(ctx context.Context, url *url.URL, params aiRequestPa
 			time.Sleep(5 * time.Second)
 		}
 	}()
+
+	// MediaMTX Output ffmpeg process
+	go func() {
+		defer func() {
+			rMediaMTX.Close()
+			if rec := recover(); rec != nil {
+				// panicked, so shut down the stream and handle it
+				err, ok := rec.(error)
+				if !ok {
+					err = errors.New("unknown error")
+				}
+				clog.Errorf(ctx, "LPMS panic err=%v", err)
+				params.liveParams.stopPipeline(fmt.Errorf("LPMS panic %w", err))
```

Comment:
Could we add a graph to our AI dashboard in grafana to track this?

Reply:
I'll raise a ticket for that 👍